components/staging/staging.html

```diff
@@ -13,10 +13,14 @@
         <textarea class="form-control" rows="2" placeholder="Body" data-bind="value: commitMessageBody, valueUpdate: 'afterkeydown', enable: !inRebase(), event: {keypress: onAltEnter}"></textarea>
         <div>
           <span class="amend" data-bind="visible: canAmend">
-                  <span class="checkmark" data-bind="css: { checked: amend }">&#10003;</span>
-                          <a href="#" data-bind="click: toggleAmend">Amend last commit</a>
-                  </span>
-                  <span class="commit-message-title-counter" data-bind="text: commitMessageTitleCount"/>
+            <div>
+              <span class="checkmark" data-bind="css: { checked: amend }">&#10003;</span>
+                <a href="#" data-bind="click: toggleAmend">Amend last commit</a>
+              </span>
+              <span class="commit-message-title bootstrap-tooltip" data-bind="text: '#' + commitMessageTitleCount()" data-toggle="tooltip" data-placement="bottom" data-original-title="# of characters in commit message" data-delay='{"show":"1000", "hide":"0"}'/>
+              <span class="commit-message-title bootstrap-tooltip" data-ta-container="ahead" data-bind="text: '&#9650;' + ahead().length"  data-toggle="tooltip" data-placement="bottom" data-original-title="# commits current is ahead compare to origin/master" data-delay='{"show":"1000", "hide":"0"}'/>
+              <span class="commit-message-title bootstrap-tooltip " data-ta-container="behind" data-bind="text: '&#9660;' + behind().length"  data-toggle="tooltip" data-placement="bottom" data-original-title="# commits current is behind compare to origin/master" data-delay='{"show":"1000", "hide":"0"}'/>
```

Comment:
Nit: Space after `bootstrap-tooltip` in `class`-attribute. (This does not exist in the lines above.)